src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+       href="#logo"
+       class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo_image"
+          alt="Moyo Company Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_link">
+          <li class="nav_item">
+            <a
+              class="nav_list is-active"
+              href="#Apple"
+            >
```

Comment:
The class `nav_list` seems to be more suitable for a list, not for a link. Consider changing it to `nav__link` or similar.